scss/header.scss

```diff
@@ -26,11 +26,47 @@
 	align-content: center;
 }
 
+/*.header__dual__logo {
+
+}*/
+
 .logo__img {
 	height: 3rem;
 	transition: all 0.5s ease;
 
 	.scrolled & {
 		height: 2rem;
 	}
+}
+
+.logo__img__1 {
```

Comment:
These class names don't convey much meaning -- how about if we take advantage of the fact that we already put the `scrolled` class on the `body` element when the page is scrolled and put any CSS for the scrolled logo there? (Check out the selector just above this one.)